### ISSUE TITLE
hello :)

### ISSUE THREAD

**Issue by DidrikxD** [MRDIKORYTB/esx_vehiclelock-double-des-cl-#1]
i cant to get it to work :( the locksmith dont work :/

**Comment by FeaReDGlock** [comment]
If you want in the main client you have [[ and ]] you juste have to delete them and its good :)

**Comment by FeaReDGlock** [comment]
You just delete the [[ and the ]]

**Comment by DidrikxD** [comment]
ahhhh, thank you so much :) have some problems with the vehicle shop, where do i put the TriggerServerEvent('esx_vehiclelock:registerkey', vehicleProps.plate, GetPlayerServerId(closestPlayer))

its says where in the discription but think my vehicle shop is diffrent from his i think :)

**Comment by FeaReDGlock** [comment]
When you sax this line : TriggerServerEvent('esx_vehicleshop:setVehicleOwned', vehicleProps) You make this line under : TriggerServerEvent('esx_vehiclelock:registerkey', vehicleProps.plate, GetPlayerServerId(closestPlayer))

**Comment by DidrikxD** [comment]
okei thank you so much again :)

**Comment by DidrikxD** [comment]
the locksmith still dont work :(

**Comment by FeaReDGlock** [comment]
Here its the code main client without problem.

**Comment by DidrikxD** [comment]
am srry for asking you so much but where does the key get? its not in my inv :) or did i read this wrong cuz i thought i could give away a key set, so others can lock my car :)

**Comment by FeaReDGlock** [comment]
I don't know too, i asked him but no reponse, i think you have to add it your self to the personal menu ;)

**Comment by DidrikxD** [comment]
ahhh okei thank you so much :)

**Comment by FeaReDGlock** [comment]
No problem. If i think how to add it i will told you

**Comment by DidrikxD** [comment]
thank you so much :)

**Comment by DidrikxD** [comment]
found a weird bug, when other ppl buy cars i get the keys :)

**Comment by DidrikxD** [comment]
yh am geting the key and not them xD

**Comment by FeaReDGlock** [comment]
Go look his github, and his mb_menuperso, you have the code for the menu key :)

**Comment by DidrikxD** [comment]
rly?!

**Comment by DidrikxD** [comment]
just download that and you can give key? :)

**Comment by FeaReDGlock** [comment]
No, you take the lines with the key, you add it to yourself nb_menuperso and done.

Because if you modified your nb_menuperso and you copy paste his files, you will lost all what you done !

You have 2 lines to add to your's files ! Its easy !

**Comment by DidrikxD** [comment]
if data2.current.value == 'menuperso_moi_clefs' then
                                TriggerEvent("esx_menu:key")
                            end

this one?

**Comment by FeaReDGlock** [comment]
Yep this one and this one : table.insert(elements, {label = 'Mes Clés',                            value = 'menuperso_moi_clefs'})

**Comment by DidrikxD** [comment]
thanks does it work to give key tho? :)

**Comment by DidrikxD** [comment]
now* :)

**Comment by FeaReDGlock** [comment]
Yeah for me its work, but when you give the key, its only the key and not the car :)
